apps/src/applab/applab.js

```diff
@@ -711,6 +719,15 @@ Applab.init = function(config) {
 
   // Applab.initMinimal();
 
+  // Ignore the user's levelHtml for levels without design mode. levelHtml
+  // should never be present on such levels, however some levels do
+  // have levelHtml stored due to a previous bug. HTML set by levelbuilder
+  // is stored in startHtml, not levelHtml.
+  // TODO(dave): remove this check once design mode content is separated
+  // from divApplab: https://www.pivotaltracker.com/story/show/103544608
+  if (level.hideDesignMode) {
+    level.levelHtml = '';
+  }
```

Comment:
Are we sure we won't ever want to have levels where levelbuilders provide some "design mode" html, but don't want to allow the users to modify that - i.e. design mode is hidden, but we still have levelHtml we want to show